integration/docker/inspect_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package docker
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+)
+
+func inspectFormatOptions(formatOption string) TableEntry {
+	return Entry(fmt.Sprintf("inspect with %s", formatOption), formatOption)
+}
+
+var _ = Describe("inspect", func() {
+	var (
+		id string
+	)
+
+	BeforeEach(func() {
+		id = randomDockerName()
+		_, _, exitCode := dockerRun("-d", "--name", id, Image)
+		Expect(exitCode).To(Equal(0))
+	})
+
+	AfterEach(func() {
+		Expect(RemoveDockerContainer(id)).To(BeTrue())
+		Expect(ExistDockerContainer(id)).NotTo(BeTrue())
+	})
+
+	DescribeTable("inspect with docker",
+		func(formatOption string) {
+			stdout, _, _ := dockerInspect("--format", formatOption, id)
+			Expect(stdout).To(ContainSubstring(Image))
+		},
+		inspectFormatOptions("'{{.Config.Image}}'"),
```

Comment:
For inspect we should try to test something related with our kata. 

I see HostConfig.Runtime would be something great for us.